check.cloudbuild.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
+- name: 'google/cloud-sdk:alpine'
+  entrypoint: 'bash'
+  args:
+  - '-eEuo'
+  - 'pipefail'
+  - '-c'
+  - |-
+    apk add --update shellcheck && rm -rf /var/chache/apk/*
+    shopt -s globstar; shellcheck -o all **/*.sh
+    docker build -t ${_DOCKER_IMAGE} .
+    git clone ${_TEST_REPO} "$$HOME/src"
+    docker run --rm -v "$$HOME/src":/src ${_DOCKER_IMAGE} cast --skipUpload --sourceDir /src
+  env:
+  - 'DOCKER_BUILDKIT=1'
```

Comment:
I just saw, in learning about buildkit, it says, this env var is not required to use buildx. - https://docs.docker.com/buildx/working-with-buildx/#build-with-buildx

I take it this is needed for something else then!

Reply:
We build test Docker container image using `docker build`, but it fails to parse our `Dockerfile` (because of this line https://github.com/tidalmigrations/cast-highlight-docker/blob/multi-arch/Dockerfile#L1) without buildkit.